### PR TITLE
Fix unknow header and add info on `register_commands`

### PR DESCRIPTION
**Pull request by Jerry-py**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -7,10 +7,8 @@ Welcome to the offical documentation of `disgames`! You can find our source code
 
 If you have any questions, check out our [faq](./faq.md)
 
-- [Disgames](#disgames)
-  - [Contents](#contents)
-  - [Installation](#installation)
-  - [Extra](#extra)
+- [Installation](#installation)
+- [Extra](#extra)
```
Comment (MarzaElise) [inline docs/index.md]: This is to follow convention and link to **all** anchor tags in the file. Revert this changes

File: docs/setups/basicbotsetup.md [hunk 2]
```diff
@@ -42,14 +42,22 @@ If you don't know what you are doing, you might wanna read everything
     !!! warning
         Please do not leave `your bot prefix` or `your token here` in the code. Please replace it with your bot prefix and your token.
 
+    !!! tip inline end
+        Stockfish path is optional. If you want to input it. Please choose stockfish as stockfish_20011801_32bit.exe. You can rename it to that
+
+        Ignore only takes in a list of commands you want to ignore. E.g. ignore=[Chess] Where Chess is imported like
```
Comment (MarzaElise) [inline docs/setups/basicbotsetup.md]: Duplicate line?

File: docs/setups/basicbotsetup.md [hunk 3]
```diff
@@ -42,14 +42,22 @@ If you don't know what you are doing, you might wanna read everything
...
+
     ```python
     from discord.ext import commands
     from disgames import register_commands
     import discord
 
     client = commands.Bot(command_prefix='your bot prefix', intents=discord.Intents.all())
 
-    register_commands(client)
+    register_commands(client, ignore=[], stockfish_path=None)
```
Comment (MarzaElise) [inline docs/setups/basicbotsetup.md]: why `ignore` and `stockfish_path` kwargs? im pretty sure it dont matter if we have it or not...

**Review by MarzaElise — CHANGES_REQUESTED** [review]

File: docs/index.md (hunk 1 above)
Comment (MarzaElise) [reply docs/index.md]: wait a sec

Comment (MarzaElise) [reply docs/index.md]: ya ok, just remove the contents part... just have it like 
- [Disgames](#disgames)
    - [Installation](#installation)
    - [Extra](#extra)

Comment (Jerry-py) [reply docs/index.md]: K

File: docs/setups/basicbotsetup.md (hunk 2 above)
Comment (Jerry-py) [reply docs/setups/basicbotsetup.md]: Whoops

File: docs/setups/basicbotsetup.md (hunk 3 above)
Comment (Jerry-py) [reply docs/setups/basicbotsetup.md]: https://github.com/andrewthederp/Disgames/blob/71bb3c5831eee98a0e07819cd263a978c1dc7e2a/disgames/__init__.py#L8

File: docs/index.md [hunk 4]
```diff
@@ -3,20 +3,19 @@
...
-  - [Contents](#contents)
   - [Installation](#installation)
   - [Extra](#extra)
 
 ## Installation
 
 To install it from [pypi](https://pypi.org/project/disgames), run
 
-```sh
+```python
```
Comment (MarzaElise) [inline docs/index.md]: this should be sh tho, its a shell command

**Review by MarzaElise — APPROVED** [review]

**Comment by MarzaElise** [comment]
everything looks good, just waiting for final review by @andrewthederp  before merding :+1:

File: docs/index.md (hunk 4 above)
Comment (Jerry-py) [reply docs/index.md]: alr will fix but no difference

**Comment by Jerry-py** [comment]
Ok

**Pull request merged**